www/assets/js/components/SearchPage.tsx

```diff
@@ -135,6 +135,7 @@ export default function SearchPage() {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const toggleResourceSearch = useCallback(
     toggleOn => async () => {
+      if (isActiveFacet(toggleOn)) return
```

Comment:
Resolved.
@alicewriteswrongs shouldn't there be a function for checking this in course-search-utils as well.Since setting active facets in the repressibility of toggleFacets imported from course-search-utils.

Reply:
possibly we could have a little helper function, sure

I think adding one over here is going to be fine for now though